client/src/pages/Reviews.js

```diff
@@ -1,25 +1,235 @@
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import { theme } from "../themes/theme";
+import ListItem from "@material-ui/core/ListItem";
+import List from "@material-ui/core/List";
+import { CardActionArea, Paper, Divider, TextField } from "@material-ui/core";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+
+// const drawerWidth = 360;
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    minWidth: "100%",
+  },
+  reviewWindow: {
+    margin: "auto",
+  },
+  reviewTitles: {
+    display: "grid",
+    textAlign: "center",
+    flexDirection: "row",
+  },
+  reviews: {
+    margin: theme.spacing(2),
+  },
+  reviewPanel: {
+    margin: theme.spacing(6),
+    textAlign: "center",
+  },
+  link: {
+    textDecoration: "none",
+  },
+  background: {
+    backgroundColor: "#ecf0fa",
+  },
+}));
 
 const Reviews = ({ state, dispatch }) => {
+  const classes = useStyles();
+  const [selectedIndex, setSelectedIndex] = React.useState(false);
+  const [reviews, setReviews] = React.useState(false);
+
+  const fetchData = async () => {
+    const result = await getReviews(dispatch);
+
+    if (result.status != 201) {
+      return false;
+    } else {
+      setReviews(result.reviews);
+    }
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const ReviewPanel = (props) => {
+    const [fetchedReview, setFetchedReview] = React.useState();
+    let { reviewId } = useParams();
+
+    const loadReview = async (reviewId) => {
+      try {
+        const result = await getReview(reviewId, dispatch);
+
+        if (result.status != 201) {
+          return false;
+        } else {
+          setFetchedReview(() => result);
+        }
+      } catch (error) {
+        console.error(error.message);
+      }
+    };
+
+    useEffect(() => {
+      loadReview(reviewId);
+    }, []);
+
+    return fetchedReview ? (
+      <Grid item classNames={classes.reviewPanelContent}>
+        <Typography component="h1" variant="h3">
+          {fetchedReview.review.title}
+        </Typography>
+        <Divider />
+        <Button>Submit</Button>
+        <Typography color="textSecondary" gutterBottom>
+          {dateToYMD(new Date(fetchedReview.review.reviewCreatedDate))}
+        </Typography>
+        <TextField></TextField>
+      </Grid>
+    ) : (
+      <Typography component="h1" variant="h3">
+        Invalid Review
+      </Typography>
+    );
+  };
+
+  const handleListItemClick = (event, index, props) => {
+    setSelectedIndex(index);
+  };
+
+  function dateToYMD(date) {
+    var strArray = [
+      "Jan",
+      "Feb",
+      "Mar",
+      "Apr",
+      "May",
+      "Jun",
+      "Jul",
+      "Aug",
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec",
+    ];
+    var d = date.getDate();
+    var m = strArray[date.getMonth()];
+    var y = date.getFullYear();
+    return "" + (d <= 9 ? "" + d : d) + " " + m + " " + y;
+  }
```

Comment:
This should be written in a utils file and not in the component itself

Reply:
I will move this code later tonight.

Reply:
Moved